flow/project.py

```diff
@@ -3600,11 +3785,8 @@ def operation_function(job):
         except KeyError:
             raise KeyError("Unknown operation '{}'.".format(args.operation))
 
-        if getattr(operation_function, '_flow_aggregate', False):
```

Comment:
What is this replacing? I'm not sure I know what the old code was meant for.

Reply:
I too wasn't sure how to go about that because I didn't find any code that sets the `_flow_aggregate` attribute for an operation.

Reply:
@csadorf It looks like you wrote this line around version 0.6.0. I don't see any other code referencing it, so I think it must be safe to delete?

Reply:
I don't know how this code snuck onto master, but it was added as part of the original push for aggregation, see [this line](https://github.com/glotzerlab/signac-flow/pull/52/files#diff-bc465f25a5bed755976338a6fd0e34ceR186) from #52. At the time we were working actively on getting the decorator syntax etc into shape, so this line probably accidentally made its way into one of those PRs.

Reply:
Looks like left-over code, probably safe to delete.